spark8t/cli/pyspark.py

```diff
@@ -44,12 +45,20 @@ def main(args: Namespace, logger: Logger):
         service_account=service_account,
         kube_interface=kube_interface,
         defaults=defaults,
-    ).pyspark_shell(args.conf, args.properties_file, extra_args)
+    ).pyspark_shell(
+        args.conf, args.properties_file, extra_args, args.ignore_configuration_hub
```

Comment:
As discussed, we can probably keep the previous UX, and set the configuration hub property to empty with a simple if statement:

```
if args.ignore_configuration_hub:
    service_account.configuration_hub = Property.empty()
```